Repositories.Tests/EmbeddedProjectRepositoryTest.cs

```diff
@@ -115,15 +116,131 @@ public async Task GetEmbeddedProject_goodflow([EmbeddedDataSource]EmbeddedProjec
         }
 
         /// <summary>
-        /// Gets the embedded projects no project.
+        /// Gets the embedded projects no embeddedProject.
         /// </summary>
-        /// <param name="project">The project.</param>
+        /// <param name="project">The embeddedProject.</param>
         [Test]
         public async Task GetEmbeddedProjects_noProject([EmbeddedDataSource]EmbeddedProject project)
         {
             // Test
             Assert.IsNull( await Repository.GetEmbeddedProjectAsync(project.Guid));
         }
+
+        /// <summary>
+        /// Finds the asynchronous user is public true.
+        /// </summary>
+        /// <param name="project">The embeddedProject.</param>
```

Comment:
The documentation in this class (mostly the parameter documentation) is very vague. Please add a proper description of what the parameters do.